### DISCUSSION TITLE
How to place a bollard?

### DISCUSSION THREAD

**Discussion by Maschga** [streetcomplete/StreetComplete#5752] [Q&A] [answered]
Hi,
I'm wondering whether it is possible to place a bollard. I can't find a way to do so:

https://github.com/user-attachments/assets/c68d4f8e-f3ef-4f63-9d06-01e9baf64df6

I'm using the version `v58.2`.

Thank you in advance!
~ Maschga

**Comment by mnalis** [comment, marked as answer]
It's not possible currently in vanilla StreetComplete.

See https://github.com/streetcomplete/StreetComplete/issues/5022#issuecomment-2081119876 and below for possible problems / reasons why, and related discussion
